### PR TITLE
Updated create_stack mock.  Response should be nestled in a stack object

### PR DESCRIPTION
**Pull request by davejohnston**
Updated Mock for create_stack
Adding basic Mock for validate_template

Tested against Openstack Kilo

### CODE REVIEW

**Comment by geemus** [comment]
@dhague @Ladas @TerryHowe - could one of you review/comment? Thanks!

File: lib/fog/openstack/requests/orchestration/create_stack.rb [hunk 1]
```diff
@@ -65,8 +65,11 @@ def create_stack(arg1, arg2 = nil)
           response = Excon::Response.new
           response.status = 201
           response.body = {
-            'id' => stack_id,
-            'links'=>[{"href"=>"http://localhost:8004/v1/fake_tenant_id/stacks/#{options[:stack_name]}/#{stack_id}", "rel"=>"self"}]}
+            'stack' = {
```
Comment (Ladas) [inline lib/fog/openstack/requests/orchestration/create_stack.rb]: @dhague is right, it should be  'stack' => {

File: lib/fog/openstack/requests/orchestration/validate_template.rb [hunk 2]
```diff
@@ -11,6 +11,14 @@ def validate_template(options = {})
           )
         end
       end
+      class Mock
+        def validate_template(options = {})
+          Excon::Response.new(
+            :body   => { 'description' => "Valid Stack"},
```
Comment (Ladas) [inline lib/fog/openstack/requests/orchestration/validate_template.rb]: response here should have 

{"Description" => ..., "Parameters" => ... , "ParameterGroups" => ...}

**Comment by plribeiro3000** [comment]
@davejohnston I know i'm late here but could you fix your git so that you can take ownership of the work you just did?

Thanks!

**Comment by dhague** [comment]
FYI, Travis is failing with this:
`        Fog::Orchestration[:openstack] | stack requests (openstack)
      - has proper format
        {"stack"=>{"id"=>"2cb72b970260c299decd40774f68a7ec", "links"=>[{"href"=>"http://localhost:8004/v1/fake_tenant_id/stacks/teststack/2cb72b970260c299decd40774f68a7ec", "rel"=>"self"}]}} does not match {"id"=>String, "links"=>Array}`

**Comment by davejohnston** [comment]
Thanks,

I've updated the test that was failing.
I can't see how to 'fix' my git, so that I can assign myself to the request.

What am I missing?

Thanks

On Nov 04, 2015, at 03:12 am, Paulo Henrique Lopes Ribeiro <notifications@github.com> wrote:

@davejohnston I know i'm late here but could you fix your git so that you can take ownership of the work you just did?
Thanks!

**Comment by plribeiro3000** [comment]
Maybe you have to add the email you are using in your local git to your config at github. Did you try that?
After that you will need to recommit and `push --force`.

**Comment by plribeiro3000** [comment]
@davejohnston I see that you fixed your git config and the build.
Unfortunately i did not get back into this in time.
Could you reopen this against `fog-openstack` please?

Thanks!

**Comment by davejohnston** [comment]
Hello, 

There is something really  great  I wanted to show  you,just take a look at that cool stuff <http://driana.fullerscustomcamo.com/lnoh/4>


dave.johnston

**Comment by Temikus** [comment]
<REDACTED SPAM from davejohnston@>

**Comment by Temikus** [comment]
REDACTED SPAM from @davejohnston 
Please, check your email account as it may have been compromised.

**Pull request closed without merging**